client/components/domains/domain-suggestion/style.scss

```diff
@@ -222,6 +222,10 @@
 	}
 }
 
+.domain-registration-suggestion__domain-title-tld {
+	word-break: break-word;
```

Comment:
I reproduced this issue on the blog onboarding flow using Safari, but `word-break: break-word;` didn't seem to fix it, but `word-break: normal;` did. Do you see the same thing?

Reply:
I don't remember if I was debugging with Firefox or Chrome. :-) This was very much fly-by fix when I noticed it was broken.

@DustyReagan mind adding `normal` instead if that worked better for you, and shipping?